Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -70,6 +72,9 @@ internal sealed class PythonProfilingPackage : AsyncPackage {
         internal static readonly string PerformanceFileFilter = Strings.PerformanceReportFilesFilter;
         private AutomationProfiling _profilingAutomation;
         private static OleMenuCommand _stopCommand, _startCommand;
+#if EXTERNAL_PROFILER_DRIVER
+        private const string ExternalProfilerDriverExe = "ExternalProfilerDriver.exe";
```

Comment:
Were there any discussions on the naming of this executable? @zooba
It has a generic name but its implementation is specific to VTune.
It's also not using the "Microsoft.PythonTools" like we do for our analyzer executable. I understand not using the prefix for executables that the user interacts with directly (less typing).

Reply:
I'm thinking Microsoft.PythonTools.Profiling.VTuneDriver.exe

Reply:
Works for me, I'll make the change

Reply:
Actually, if we could postpone this change 'til after the merge, it'd be very useful.  Changing the name of the executable involves changes in the signing and skip validation infrastructures that I'm not sure I can make on my own.  I've added a comment in 50e290aa5 so that this change does not fall through the cracks.

Reply:
Also, the idea is that ExternalProfilerDriver becomes a component that knows how to deal with more than one profiler, parses its input and creates diagsessions.  VTune is just the first one.  That was the motivation for the name.  I'm still confident that we can get teach it of at least one more profiler.

Reply:
Agreed to postpone.